ui/src/app/pages/admin/admin-notebook-view.tsx

```diff
@@ -135,11 +136,11 @@ const AdminNotebookView = (spinnerProps: WithSpinnerOverlayProps) => {
   const { ns, nbName } = useParams<MatchParams>();
```

Comment:
`nbName` always has the suffix!  So we are good here - confirmed by testing on Test